test/Winton.DomainModelling.DocumentDb.IntegrationTests/EntityFacadeFactoryTests.cs

```diff
@@ -14,30 +14,31 @@
 
 namespace Winton.DomainModelling.DocumentDb
 {
-    public class EntityFacadeTests : IDisposable
+    public class EntityFacadeFactoryTests : IDisposable
```

Comment:
In a similar vein to the integration test for `ValueObjectFacade` I think this should be named `EntityFacadeTests`.